ccloud/README.md

```diff
@@ -1,19 +1,44 @@
 ![image](../images/confluent-logo-300-2.png)
 
+* [Overview](#overview)
+* [Beginner Demo](#beginner-demo)
+* [Advanced Demo](#advanced-demo)
+* [Client Code Examples](#client-code-examples)
+* [Auto-generate Configurations to connect to Confluent Cloud](#auto-generate-configurations-to-connect-to-confluent-cloud)
+
+
 # Overview
 
-This Confluent Cloud demo showcases a hybrid Kafka deployment: one cluster is a self-managed cluster running locally, the other is a |ccloud| cluster.
-Data streams into topics both a local cluster and a cluster in Confluent Cloud, and Replicator copies the on-prem data to Cloud so that stream processing can happen in the Cloud.
+[Confluent Cloud](https://docs.confluent.io/current/cloud/index.html) is a resilient, scalable streaming data service based on Apache Kafka®, delivered as a fully managed service.
+It has a web interface and local command line interface that you can use to manage cluster resources, Kafka topics, Schema Registry, and other services.
 
-Note: if you are looking for code examples of producers writing to and consumers reading from Confluent Cloud, or producers and consumers using Avro with Confluent Schema Registry, checkout the [clients subdirectory of examples](../clients). It provides client examples written in various programming languages.
+This repo has a few resources to help you validate your solutions on Confluent Cloud.
 
-This automated demo is an expansion of the [KSQL Tutorial](https://docs.confluent.io/current/ksql/docs/tutorials/basics-local.html#create-a-stream-and-table>). Instead of the Kafka cluster backing the KSQL stream processing running on your local install, it runs on your Confluent Cloud cluster. There are also additional Confluent Platform components including Confluent Control Center and Confluent Replicator.
+# Beginner Demo
+
+[This beginner demo](../security/acls/acl.sh) is a fully scripted demo that shows users how to interact with Confluent Cloud using the CLI.
```

Comment:
Should we redo the script names and paths as well to head off any confusion?

Reply:
Yes we should.  

The reason I hadn't up till now is because there are docs links that would break if we just eliminated it.  However, I think a soft link would work.  I'll validate and update.